docs/source/docs/hardware/selecting-hardware.md

```diff
@@ -41,6 +41,8 @@ For colored shape detection, any non-fisheye camera supported by PhotonVision wi
 
 For driver camera, we recommend a USB camera with a fisheye lens, so your driver can see more of the field.
 
+For object detection, we recommend an Arducam USB OV9782 100fps color camera.
```

Comment:
The requirements for object detection aren't this high. General opinion seems to be that any color webcam with 640x480 resolution should work fine as long as it is supported by v4l2. Wide angle lenses should also be OK since distortion shouldn't be much of a concern. 

Would you update this paragraph with a more general recommendation? After that, you can use the Arducam as an example of one that is known to work. Another one that is known to work is the Microsoft Lifecam 3000 that a lot of teams have from previous years. Maybe this should also be added as another recommendation.

Reply:
We had a previous PR open that was also to add a suggestion. I'd like to make it clear just about any color camera works fine.

Reply:
I think this should be completely removed. The point was to remove "recommended" hardware from the selecting hardware section and put it in the quick-start. It just doesn't say that wide angle also works. We could also recommend the hd3000 but I wasn't sure cause it had some weirdnesses.

Reply:
Ok, I think this is my fault. I had made an earlier fork and added this line in an earlier PR. The reason was because I was buying cameras for the OPi5, and I asked for the best recommendation on Discord. 
I thought I had reverted my fork back to the master. I failed to do that and this line ended up in PR1591.
At this point, I am not sure how to proceed. Is the finalization of the PR someone else's responsibility? Should I edit this? Delete it? I am open to instruction. 🙂

Reply:
For the change to this file, it looks like the consensus is to remove the line. You can edit it in the web interface by clicking on the “Files Changed” tab at below the PR title and then selecting the Edit File option on this file. 
![image](https://github.com/user-attachments/assets/9e20e848-c62a-4d61-938b-f2bc1965e6bb)

When done, you can commit the edits.